android/app/src/main/java/fi/fmi/mobileweather/UpdateWidgetJobIntentService.java

```diff
@@ -166,16 +171,37 @@ protected void updateCoordinates(double latitude, double longitude, int widget_i
         latitude = (double)Math.round(latitude * 10000d) / 10000d;
         longitude = (double)Math.round(longitude * 10000d) / 10000d;
 
-        String params = "locations="+latitude+","+longitude+"&l="+language
-                +"&version="+version;
+        String latlon = latitude+","+longitude;
+        /*String params = "locations="+latitude+","+longitude+"&l="+language
+                +"&version="+version;*/
 
         //Log.d("params", params);
 
         // Large widget
 
-        String url =  "https://widget.weatherproof.fi/android/androidwidget.php?"+params;
+        // forecast url
+        String url = "https://data.fmi.fi/fmi-apikey/ff22323b-ac44-486c-887c-3fb6ddf1116c/timeseries?latlon=" +
```

Comment:
Do not store URL with apikey to source code, read it from config

Reply:
UpdateWidgetJobIntentService changes can be ignored because that file will be removed soon. :)

Reply:
We can't reveal apikeys in source code, because somebody can copy those and use in own projects. Also ChatGPT and other AI clients  can use this for learning and suggest apikeys for users. So not even temporarily.

Reply:
File removed